src/rgw/store/sfs/sqlite/versioned_object/versioned_object_definitions.h

```diff
@@ -42,6 +43,7 @@ struct DBOPVersionedObjectInfo {
   ceph::real_time creation_time;
   ObjectState object_state;
   std::string version_id;
+  std::string etag;
```

Comment:
Could you add this new variable to the unit tests?
It's so we don't loose coverage and test all the members.

Something like the following should be enough:
```diff
diff --git a/src/test/rgw/sfs/test_rgw_sfs_sqlite_versioned_objects.cc b/src/test/rgw/sfs/test_rgw_sfs_sqlite_versioned_objects.cc
index 8851597d6e..be30f9164f 100644
--- a/src/test/rgw/sfs/test_rgw_sfs_sqlite_versioned_objects.cc
+++ b/src/test/rgw/sfs/test_rgw_sfs_sqlite_versioned_objects.cc
@@ -116,6 +116,7 @@ DBOPVersionedObjectInfo createTestVersionedObject(uint id, const std::string & o
   test_versioned_object.creation_time = ceph::real_clock::now();
   test_versioned_object.object_state = rgw::sal::ObjectState::OPEN;
   test_versioned_object.version_id = "test_version_id_" + suffix;
+  test_versioned_object.etag = "test_etag_" + suffix;
   return test_versioned_object;
 }
 
@@ -128,6 +129,7 @@ void compareVersionedObjects(const DBOPVersionedObjectInfo & origin, const DBOPV
   ASSERT_EQ(origin.creation_time, dest.creation_time);
   ASSERT_EQ(origin.object_state, dest.object_state);
   ASSERT_EQ(origin.version_id, dest.version_id);
+  ASSERT_EQ(origin.etag, dest.etag);
 }
 
 TEST_F(TestSFSSQLiteVersionedObjects, CreateAndGet) {
```

Reply:
done